### DISCUSSION TITLE
remove devices from the Casambi Bluetooth integration

### DISCUSSION THREAD

**Discussion by Alexei-vk2** [lkempf/casambi-bt-hass#77] [Show and tell]
Hello!
how do I remove devices from the Casambi Bluetooth integration?

![Снимок экрана 2024-04-11 211904](https://github.com/lkempf/casambi-bt-hass/assets/166062625/daea6f48-6b6b-4c60-bb45-51832e62615b)

**Comment by lkempf** [comment]
You can disable devices in HA so that they aren't shown. To do this just click the edit button when viewing a device.

**Comment by Alexei-vk2** [comment]
Hello!
Faced with this situation:
Casambi integration is installed in the home assistant.
when the "unavailable" condition occurs, the lights turn on and off.

is there any way to make sure that even after the "not available" state, the devices have the last mode so that they are turned off?
can you tell me how to do this?
![WhatsApp Image 2024-05-13 at 10 57 27](https://github.com/lkempf/casambi-bt-hass/assets/166062625/4df67383-01e7-49c7-bd81-e5d1ad2e29f9)
